segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -425,6 +438,8 @@ boolean isStorageSystemSegment(SegmentMetadata segmentMetadata) {
         checkInitialized();
         return executeSerialized(() -> {
             val traceId = LoggerHelpers.traceEnter(log, "seal", handle);
+            Timer timer = new Timer();
+            log.debug("{} seal - segment={} latency={}.", logPrefix, handle.getSegmentName());
```

Comment:
`seal - started` to be consistent with the rest of the log messages you added.

Reply:
Thanks. I missed that